skywalking/plugins/sw_mysqlclient.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from skywalking import Layer, Component, config
+from skywalking.trace.context import get_context
+from skywalking.trace.tags import TagDbType, TagDbInstance, TagDbStatement, TagDbSqlParameters
+
+link_vector = ['https://mysqlclient.readthedocs.io/']
+support_matrix = {
+    'mysqlclient': {
+        '>=2.1.0': ['1.0']
+    }
+}
+note = """"""
+
+
+def install():
+
+    from MySQLdb.cursors import Cursor
+
+    _execute = Cursor.execute
+
+    def _sw_execute(this: Cursor, query, args=None):
+        peer = f'{this.connection.get_host_info().split()[0]}:{this.connection.port}'
+        context = get_context()
+        with context.new_exit_span(op='Mysql/MysqlClient/execute', peer=peer, component=Component.MysqlClient) as span:
+            span.layer = Layer.Database
+            _execute(this, 'select database()')
```

Comment:
I'd rather not to execute sql just for the database name.

Did you read this https://github.com/apache/skywalking-python/pull/178#discussion_r793186936 ?

Reply:
@kezhenxu94 we change the method to intercept